bot/seasons/valentines/lovecalculator.py

```diff
@@ -0,0 +1,71 @@
+import json
+import logging
+from pathlib import Path
+from random import choice
+
+import discord
+from discord.ext import commands
+
+from bot.constants import Roles
+
+log = logging.getLogger(__name__)
+
+
+class LoveCalculator:
+    """
+    A cog for calculating the love between two people
+    """
+    def __init__(self, bot):
+        self.bot = bot
+
+    @commands.command(aliases=('love_calculator', 'love_calc'))
+    @commands.cooldown(rate=1, per=5.0, type=commands.BucketType.user)
+    async def love(self, ctx, name_one: discord.Member, name_two=None):
```

Comment:
`name_two` should still be a converter, as it'll only convert when an arg is passed to it.

```suggestion
    async def love(self, ctx, name_one: discord.Member, name_two: discord.Member = None):
```

Reply:
Added. I didn't even know this was possible o.o